modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -223,15 +223,14 @@ For more information, see <<clustering-discovery-dns>>.
 It is important that both `dbms.kubernetes.service_port_name` and `dbms.kubernetes.discovery.v2.service_port_name` are set during the operation.
 For more information, see <<clustering-discovery-k8s>>.
 
-[[discovery-v1-to-v2-in-place]]
-=== In-place rolling
+[[discovery-v1-to-v2-semi-automated-procedure]]
+=== Moving whole cluster with a procedure
```

Comment:
```suggestion
=== Procedure for moving an entire cluster
```

Reply:
To align the style of the first and second titles, we can make them: 

- Moving an entire cluster with a procedure

- Running a procedure on each server

or 

- Procedure for moving an entire cluster

- Per server procedure

I'd prefer the second variant.

Reply:
I have changed the titles to align with the second option.